docs/infrastructure_and_maintenance/backup.md

```diff
@@ -4,7 +4,7 @@ description: Periodically back up your Repository information by making a databa
 
 # Backup 
 
-You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux-UNIX-based system where [[= product_name =]] is using a MySQL database called "example". You should shut down Platform if it's running before making a backup or an upgrade.
+You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux-UNIX-based system. You should shut down the DXP if it's running before making a backup.
```

Comment:
Actually, the example below also covers PostgreSQL. Upgrade is not mentioned in this page at all.